vision/google/cloud/vision/image.py

```diff
@@ -54,7 +54,7 @@ def __init__(self, client, content=None, filename=None, source_uri=None):
                 content = file_obj.read()
 
         if content is not None:
-            content = _bytes_to_unicode(b64encode(_to_bytes(content)))
+            content = _to_bytes(content)
```

Comment:
Sorry I was vague on the kind of encoding.

The pb code base64 encodes the bytes for us. But yeah they're always bytes.

The only place it gets converted to `str` is in the comment below which is in the `to_dict` method.